metadata-ingestion/tests/integration/metabase/test_metabase.py

```diff
@@ -245,3 +303,34 @@ def test_mode_ingest_failure(pytestconfig, tmp_path):
             assert exec_error.args[0] == "Source reported errors"
             assert len(exec_error.args[1].failures) == 1
             assert list(exec_error.args[1].failures.keys())[0] == "metabase-dashboard"
+
+
+@freeze_time(FROZEN_TIME)
+def test_9767_templated_query_is_stripped(
+    pytestconfig, tmp_path, test_pipeline, mock_datahub_graph
```

Comment:
this shouldn't need all these pytest fixtures

Reply:
done